orchestra/utils/task_lifecycle.py

```diff
@@ -549,7 +549,7 @@ def tasks_assigned_to_worker(worker):
             next_todo_title = None
             next_todo_dict = {}
             should_be_active = False
-            if state in ('returned', 'in_progress'):
+            if state in ('returned', 'in_progress', 'paused'):
```

Comment:
Wow that was simple! :) If you rested this in dev with production data, and didn't run into issues, that's great! :)

Reply:
One thing to watch out for: I assume it's OK, but do we only use `should_be_active` (set below) for `returned`/`in_progress` tasks, or do we have to be careful about how it might be used in the `paused` table?

Reply:
I'll add a guard to check if it's a paused task so that we can have the same behavior as before for `should_be_active`.

Reply:
LGTM assuming you compared on real data before/after! Thanks for programming defensively! :)